README.md

```diff
@@ -1 +1,281 @@
-# math-magicians-app
\ No newline at end of file
+<a name="readme-top">Math Magician App</a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] Please kindly remove this comment and any other code comment in your codebase

Reply:
@AnselemOdims  this is not the final project so i will need the comments for when i am done